userspace/falco/falco.cpp

```diff
@@ -137,8 +137,8 @@ static void usage()
 	   " -l <rule>                     Show the name and description of the rule with name <rule> and exit.\n"
 	   " --list [<source>]             List all defined fields. If <source> is provided, only list those fields for\n"
 	   "                               the source <source>. Current values for <source> are \"syscall\", \"k8s_audit\"\n"
-	   " --list-plugins                Print info on all loaded plugins and exit.\n"
 #ifndef MINIMAL_BUILD
+	   " --list-plugins                Print info on all loaded plugins and exit.\n"
```

Comment:
It seems to me that this commit is not implementing the wanted behavior.

Indeed, `MINIMAL_BUILD` is not the feature gate for the statically linked build. I guess you wanted to use `MUSL_OPTIMIZED_BUILD` instead :point_down: 

```
if(MINIMAL_BUILD)
  set(MINIMAL_BUILD_FLAGS "-DMINIMAL_BUILD")
endif()

if(MUSL_OPTIMIZED_BUILD)
  set(MUSL_FLAGS "-static -Os -fPIE -pie")
endif()
```

right?

Reply:
Sure, makes sense, I changed it.